### DISCUSSION TITLE
Response object in another package

### DISCUSSION THREAD

**Discussion by Dallin-Cawley** [parvez3019/go-swagger3#37] [Q&A]
I have a handler godoc comment as follows

```// UpdateTournament is the orchestrators for the update tournament route.
//
// @Title Update Resource
// @Description Updates the Resource from the provided options.
// @Param  id path  int  true  "ID of resource"
// @Param request body UpdateResourceRequest true "The options used to update the Resource"
// @Success 200 object {a different package}.Resource "ResourceJSON"
// @Resource {a different package}.Resource
// @Route /resource/{id} [patch]
func UpdateResource(ctx *gin.Context) {

}
```

With the `{a different packager}.Resource` defined as 

```
type Resource struct {
...
}
```
There is not schema created for the `{a different package}.Resource` when using the following command

`go-swagger3 --module-path . --main-file-path ./main.go --output open_api_spec.json --schema-without-pkg --generate-yaml true`

And as a result, there is no example response in the generated swagger UI. Any suggestions?

**Comment by parvez3019** [comment]
Hi @Dallin-Cawley ,

Just going by my hunch on this, you might be missing an import for that {a different package}.Resource for the package.
If there is no import statement for any package, the auto-generation doesn't kick in.

If this is the same with you, you can try to add an empty import to that package.
Example - https://github.com/parvez3019/go-swagger3/blob/main/integration_test/test_data/handler/restaurants.go

There are many other examples you can find the integrations tests which also works as examples for this.
